extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -11,9 +11,9 @@ import * as constants from './constants';
 import { BindingType } from 'sql-bindings';
 import { ConnectionDetails, IConnectionInfo } from 'vscode-mssql';
 // https://github.com/microsoft/vscode-azurefunctions/blob/main/src/vscode-azurefunctions.api.d.ts
-import { AzureFunctionsExtensionApi } from '../typings/vscode-azurefunctions.api';
+import { AzureFunctionsExtensionApi } from '../../../types/vscode-azurefunctions.api';
```

Comment:
Unfortunately these aren't declared as modules so we can't import them through ref.d.ts like vscode-mssql. It's fine to reference typings outside the extension folder though since that's only a build time thing.